### PR TITLE
GIX-2149: Remove ENABLE_CKETH flag

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Clean up after ckETH release.

Remove ENABLE_CKETH flag.

# Changes

* Remove the flag from dfx.json.
* Remove it from the env constants.
* Remove it from the feature flags store.
* Remove the usafe of `ENABLE_CKETH` store in cketh-canisters.services.ts

# Tests

## Automatic Changes

* I run the script `./scripts/nns-dapp/test-config --update` which updated "test-config-assets/app" and "test-config-assets/mainnet" files.

## Manual Changes

* Remove the instance of the flag in "test-config-assets/local" manually.
* Remove the lines that override the value of `ENABLE_CKETH` in the tests.
* Remove the tests that required `ENABLE_CKETH` to be `false`.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4302** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4302?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4302?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/services/cketh-canisters.services.spec.ts [hunk 1]
```diff
@@ -73,36 +67,5 @@ describe("cketh-canisters.services", () => {
         });
       });
     });
-
-    describe("if ckethtest is enabled", () => {
-      beforeEach(() => {
-        overrideFeatureFlagsStore.setFlag("ENABLE_CKTESTBTC", true);
-      });
-
-      it("should load ckethtest canisters if cktestbtc is enabled", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/services/cketh-canisters.services.spec.ts:82]: Shouldn't we keep this test?

Comment (lmuntaner) [reply frontend/src/tests/lib/services/cketh-canisters.services.spec.ts:82]: It's covered above: `"if ckethtest is enabled" > "should load cketh canisters"`.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**